### PR TITLE
Test PR build

### PR DESCRIPTION
**Pull request by bjornreppen**

### CODE REVIEW

**Comment by artsdatabank** [comment]
Test den: https://forvaltningsportal.test.artsdatabanken.no/pr/forvaltningsportal_409

File: database/README.md [hunk 1]
```diff
@@ -187,6 +237,51 @@ WHERE datasettkode='KOM'
...
+FROM datasett ds
+WHERE datasettkode = ANY(regexp_split_to_array(kartlag,','));
+$$ LANGUAGE SQL IMMUTABLE;
+
+
+
+curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"kartlag":"FYL", "wkt": "POLYGON ((10 60,10 65,20 60,10 60))"}' \
+  https://forvaltningsportalapi.test.artsdatabanken.no/rpc/arealstatistikk_wkt
```
Comment (kepatinin) [inline database/README.md:264]: Shouldn't we use the new URL? https://okologiskegrunnkartapi.test.artsdatabanken.no/rpc/arealstatistikk_wkt

File: database/Naturtyper.md [hunk 2]
```diff
@@ -87,14 +87,14 @@ LANGUAGE PLPGSQL;
...
-RETURNS json AS $$
+CREATE OR REPLACE FUNCTION .naturtype_5k_json(områdekid varchar)
+RETURNS json AS $$import_nin
 BEGIN
 	RETURN (SELECT to_json(sub)
 	FROM  (
       SELECT json_agg(REPLACE(kartleggingsenhetkode,'NA_','')) AS "type"
 	   FROM   import_nin.områder5k c
-	   LEFT   JOIN import_nin.kartleggingsenheter5k ct ON ct.område5kid = c.område5kid
+	   LEFT   JOIN import_nin.kartl	eggingsenheter5k ct ON ct.område5kid = c.område5kid
```
Comment (kepatinin) [inline database/Naturtyper.md:97]: Is this a typo? `kartl	eggingsenheter5k`

**Review by kepatinin — APPROVED** [review]
LGTM! 👍 just a couple of comments, but I'm not sure if they are relevant

**Comment by artsdatabank** [comment]
Test den: https://forvaltningsportal.test.artsdatabanken.no/pr/forvaltningsportal_409

**Comment by artsdatabank** [comment]
Test den: https://okologiskegrunnkart.test.artsdatabanken.no/pr/409

**Comment by artsdatabank** [comment]
Test den: https://okologiskegrunnkart.test.artsdatabanken.no/pr/409

**Comment by bjornreppen** [comment]
I think it might work now..

**Comment by artsdatabank** [comment]
Test den: https://okologiskegrunnkart.test.artsdatabanken.no/pr/409

File: database/Naturtyper.md (hunk 2 above)
Comment (bjornreppen) [reply database/Naturtyper.md]: yes, typo, thanks

**Pull request merged**